src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -0,0 +1,1665 @@
+        this->EvapEnergy = this->QEvaporator * state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;
+        this->CondEnergy = this->QCondenser * state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;
+        this->OilCoolerEnergy = this->QOilCooler * state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;
+        this->AuxiliaryEnergy = this->QAuxiliary * state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;
+        this->EvapInletTemp = state.dataLoopNodes->Node(this->EvapInletNodeNum).Temp;
+        this->CondInletTemp = state.dataLoopNodes->Node(this->CondInletNodeNum).Temp;
+        this->CondOutletTemp = state.dataLoopNodes->Node(this->CondOutletNodeNum).Temp;
+        if (this->Power != 0.0) {
+            this->ActualCOP = this->QEvaporator / this->Power;
+        } else {
+            this->ActualCOP = 0.0;
+        }
+    }
+
+    // Calculate in case of standby power
+    this->AmbientZoneGainEnergy = this->AmbientZoneGain * state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;
+    this->Energy = this->Power * state.dataHVACGlobal->TimeStepSys * DataGlobalConstants::SecInHour;
+}
+
+void ASHRAE205ChillerSpecs::simulate(
+    EnergyPlusData &state, const PlantLocation &calledFromLocation, bool FirstHVACIteration, Real64 &CurLoad, bool RunFlag)
+{
+    if (calledFromLocation.loopNum == this->CWPlantLoc.loopNum) {
+        this->initialize(state, RunFlag, CurLoad);
+        this->calculate(state, CurLoad, RunFlag);
+        this->update(state, CurLoad, RunFlag);
+    } else if (calledFromLocation.loopNum == this->CDPlantLoc.loopNum) {
+        DataPlant::LoopSideLocation LoopSide = this->CDPlantLoc.loopSideNum;
+        PlantUtilities::UpdateChillerComponentCondenserSide(state,
+                                                            calledFromLocation.loopNum,
+                                                            LoopSide,
+                                                            DataPlant::PlantEquipmentType::Chiller_ElectricASHRAE205,
+                                                            this->CondInletNodeNum,
+                                                            this->CondOutletNodeNum,
+                                                            this->QCondenser,
+                                                            this->CondInletTemp,
+                                                            this->CondOutletTemp,
+                                                            this->CondMassFlowRate,
+                                                            FirstHVACIteration);
+    }
+}
+
+void ASHRAE205ChillerSpecs::getDesignCapacities(
+    [[maybe_unused]] EnergyPlusData &state, const PlantLocation &calledFromLocation, Real64 &MaxLoad, Real64 &MinLoad, Real64 &OptLoad)
+{
+    if (calledFromLocation.loopNum == this->CWPlantLoc.loopNum) {
+        MinLoad = this->Representation->performance.performance_map_cooling
```

Comment:
Interesting.  MinLoad is usually 0.

Reply:
Maybe it should be? I was following the lead of the other chiller objects which use rated capacity times minimum part load ratio. Though it looks like a typo here. `this->MaxSequenceNumber` should be `this->MinSequenceNumber`. That's my bad. @tanaya-bigladder, can you make that change if you already have other modifications in progress?

Reply:
I will rely on you all here as you know the 205 model better than me.  I really would assume it would be 0 for minimum load, but if you think minimum load should be treated this way then so be it.

Reply:
I honestly don't know how I should be interpreting `MinLoad`. ASHRAE 205 chillers can cycle down to zero just like other chillers can below their minimum part load ratio, but the [EIR chiller doesn't use zero](https://github.com/NREL/EnergyPlus/blob/2b7eae49e2f7e8401a3904852cc200fbe87185f7/src/EnergyPlus/ChillerElectricEIR.cc#L185). Maybe they both should?

Reply:
My understanding is that `MinLoad` is only used for load allocation when there is more than one piece of equipment available. The EIR chiller `MinPartLoadRat` is a user input that defaults to 0.1. The IDD describes it as: Part load ratio below which the chiller starts cycling on/off to meet the load. So the goal is for load control to avoid putting on chiller in cycling when another one is also running. So if you have a way to search the performance map to find where it switches from continuous to cycling, then use that. Otherwise I would go with zero.

Reply:
Thanks, @mjwitte! The ASHRAE 205 performance map is fairly analogous to the multi-/variable-speed DX coils. It defines performance a several compressor speeds and cycles below the lowest speed. Based on what you have described, I believe we are handling this correctly.